### DISCUSSION TITLE
Image gets cut and patched together in weird ways

### DISCUSSION THREAD

**Discussion by rcastonguay** [ImageMagick/ImageMagick#3694] [Digital Image Processing]
Hi, all,
I'm new to ImageImagick and my tech skills are OK but not super awesome like most people here (so explain clearly!). I used ImageMagick with Wax (demo site: https://minicomp.github.io/wax/) and encountered a strange outcome. It might more a viewer (IIIF) problem though, I'm not sure. Take a look at this url to see what I mean: 
https://rcastonguay.github.io/wax-main/new-haven-pics/img_0377/ 

The derivatives were created with this command: bundle exec rake wax:derivatives:simple YOUR_COLLECTION_NAME

When I take a look at the actual derivatives on my hard drive they look fine though. Any ideas?

Rémi

**Comment by fmw42** [comment]
I do not understand your issue. The link just goes to some Wax pictures.  So what is wrong with them? They all look fine.  Perhaps  your computer is running out of RAM or your bandwidth is too low to download them to your display device.

**Reply by rcastonguay** [reply]
Do you not see how the picture got cut up in segments and kind of patched together? I’m referring to the second link in my post.

**Reply by fmw42** [reply]
No, they look fine to me. Example below:

![Screen Shot 2021-05-17 at 1 42 13 PM](https://user-images.githubusercontent.com/13458218/118553971-d921cf80-b715-11eb-89e5-a404a1871e89.png)

**Comment by snibgo** [comment]
I see a kind of cut-and-patch problem:
![x](https://user-images.githubusercontent.com/33812368/118556361-aef30080-b75b-11eb-870f-ab485d1620d0.jpg)

But know nothing about how the images are made. Is ImageMagick involved?

**Reply by rcastonguay** [reply]
Very strange. What about this one: https://rcastonguay.github.io/wax-main/new-haven-pics/img_0384/

r.

**Reply by rcastonguay** [reply]
Great, I'm glad it's not just me! Yes, I believe ImageMagick is involved. Actually I was wrong about the command I ran. This is what I did at the command line: bundle exec rake wax:derivatives:iiif YOUR_COLLECTION_NAME

You can take a look at the documentation I followed here: https://minicomp.github.io/wiki/wax/running-the-tasks/derivatives/ 

Thank you all.

**Reply by fmw42** [reply]
I see none of those artifacts using Mac Preview to view them.

**Reply by rcastonguay** [reply]
Sounds like the problem is with the IIIF viewer then?

**Reply by fmw42** [reply]
Perhaps Wax is creating a non-standard format or its viewer is not compatible with your computer OS version.

**Comment by snibgo** [comment]
> You can take a look at the documentation I followed here: https://minicomp.github.io/wiki/wax/running-the-tasks/derivatives/

I see nothing there about ImageMagick.

**Reply by fmw42** [reply]
https://minicomp.github.io/wiki/wax/system-requirements/ lists ImageMagick
